src/UKHO.S100PermitService.Common/Models/HoldingsServiceResponse.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace UKHO.S100PermitService.Common.Models
```

Comment:
Move to the Services namespace

Reply:
As discussed, models for services are moved to their appropriate namespaces.

Reply:
Resolved and moved to Holdings namespace